### PR TITLE
osd: skip dumping logical devices

### PR DESCRIPTION
**Pull request by Songweibin**
The code meant to do so but actually it does not ...

Signed-off-by: songweibin <song.weibin@zte.com.cn>

### CODE REVIEW

**Comment by xiexingguo** [comment]
This looks good to me. @tchaikov Can you double check this?

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -2337,10 +2337,10 @@ will start to track new ops received afterwards.";
     store->get_devices(&devnames);
     f->open_object_section("list_devices");
     for (auto dev : devnames) {
-      f->dump_string("device", "/dev/" + dev);
       if (dev.find("dm-") == 0) {
-	continue;
```
Comment (tchaikov) [inline src/osd/OSD.cc]: it's fine to use tab here. as tabstop = 8 in ceph.

**Comment by Songweibin** [comment]
I repushed it, thanks.

**Comment by tchaikov** [comment]
retest this please

**Pull request merged**